projects/packages/videopress/src/client/admin/components/edit-video-details/index.tsx

```diff
@@ -72,11 +73,12 @@ const Header = ( {
 };
 
 const GoBackLink = () => {
-	const history = useHistory();
+	const { page } = useVideosQuery();
+	const to = page > 1 ? `/?page=${ page }` : '/';
```

Comment:
`import { removeQueryArgs } from '@wordpress/url';` can be used to avoid relying on manually setting the parameter.